components/hab/src/cli.rs

```diff
@@ -403,6 +403,21 @@ pub fn get() -> App<'static, 'static> {
                 (@arg PKG_IDENT: +required +takes_value {valid_ident}
                     "A package identifier (ex: core/redis, core/busybox-static/1.42.2)")
             )
+            (@subcommand list =>
+                (about: "List all versions of installed packages")
+                (aliases: &["li"])
+                (@group prefix =>
+                    (@attributes +required)
+                    (@arg ALL: -a --all
+                            "List all installed packages")
+                    (@arg ORIGIN: -o --origin +takes_value
+                            "An origin to list")
+                    (@arg PKG_IDENT: +takes_value {valid_ident}
+                    "A package identifier (ex: core/redis, core/busybox-static/1.42.2).")
+
```

Comment:
What's here is completely fine, but is there any thought to having `--all` be the default, so `hab pkg list` would also work?

Reply:
We had that before, but moved towards this more structured style during review.  I'd leave it as is and get feedback from the community